src/include/btree.i

```diff
@@ -979,6 +980,13 @@ __wt_page_can_evict(WT_SESSION_IMPL *session, WT_PAGE *page, int check_splits)
 		return (0);
 
 	/*
+	 * Allow for the splitting of pages when a checkpoint is underway
```

Comment:
I'd make this comment bigger, something like:

Split big pages in memory if there is a checkpoint currently running. Don't keep splitting the same pages repeatedly.